ext/standard/math.c

```diff
@@ -389,31 +389,38 @@ PHP_FUNCTION(round)
 }
 /* }}} */
 
-#define PHP_CLAMP(zvalue, zmin, zmax) { \
-	if (EXPECTED(Z_TYPE_P(zmin) == IS_DOUBLE) && UNEXPECTED(zend_isnan(Z_DVAL_P(zmin)))) { \
-		zend_argument_value_error(2, "cannot be NAN"); \
-		RETURN_THROWS(); \
-	} \
- \
-	if (EXPECTED(Z_TYPE_P(zmax) == IS_DOUBLE) && UNEXPECTED(zend_isnan(Z_DVAL_P(zmax)))) { \
-		zend_argument_value_error(3, "cannot be NAN"); \
-		RETURN_THROWS(); \
-	} \
- \
-	if (zend_compare(zmax, zmin) == -1) { \
-		zend_argument_value_error(2, "must be smaller than or equal to argument #3 ($max)"); \
-		RETURN_THROWS(); \
-	} \
- \
-	if (zend_compare(zmax, zvalue) == -1) { \
-		RETURN_COPY(zmax); \
-	} \
- \
-	if (zend_compare(zvalue, zmin) == -1) { \
-		RETURN_COPY(zmin); \
-	} \
- \
-	RETURN_COPY(zvalue); \
+/* Return the given value if in range of min and max */
+static void _clamp(zval *return_value, zval *value, zval *min, zval *max)
```

Comment:
Please don't start function names with an underscore. Those are reserved names for C.
I know we have some in the codebase still, but we're trying to not introduce more and are getting rid of those name violations from time to time.

Reply:
Renamed, if you prefer something else, feel free to code-suggest it. 🙏